wiki/Tournaments/OWC/2015/de.md

```diff
@@ -1,215 +1,480 @@
-   2. Um erfolgreich akzeptiert zu werden, muss Ihr globaler Rang in osu! mindestens \#5000 oder besser sein.
-   3. Um erfolgreich akzeptiert zu werden, sollten Sie sichergehen, dass Sie nicht die [osu! Community Regeln](/wiki/Rules) in den letzten 12 Monaten verletzt haben.
+   - Das Turnier-Management wird eine Liste mit potentiellen Kandidaten für das jeweilige Länderteam erstellen.
+   - Das Turnier-Management erklärt einen Kandidaten als Kapitän für sein Landes Team.
+   - Der ausgewählte Kapitän kann ein Team aus einer Liste mit Kandidaten aus dessen Land erstellen.
+2. Um gültige und ernsthafte Anmeldungen zu gewährleisten, wird jede Anmeldung und Veränderung vom Turnier-Management überprüft.
+   - Jeder Spieler wird als potenzielles Teammitglied in Betracht gezogen. Die Flagge im Profil ist ausschlaggebend.
+   - Um erfolgreich akzeptiert zu werden, muss dein globaler Rang in osu! mindestens \#5000 oder besser sein.
+   - Um erfolgreich akzeptiert zu werden, solltest du sichergehen, dass du nicht die [osu! Community-Regeln](/wiki/Rules) in den letzten 12 Monaten verletzt hast.
 3. Alle erfolgreich zusammengestellten Teams werden nach der Registrierungsphase veröffentlicht.
 4. Abgelehnte Spieler können diese Entscheidung anfechten, indem sie [tournaments@ppy.sh](mailto:tournaments@ppy.sh) kontaktieren.
-5. Map Wähler dürfen nicht am Turnier als Spieler teilnehmen.
+5. Beatmap-Wähler dürfen nicht am Turnier als Spieler teilnehmen.
 
 ### Etappenanweisungen
 
 1. In der ersten Etappe (Gruppenphase) werden alle Teams in 8 Gruppen mit jeweils 4 Teams aufgeteilt.
-
 2. Die Teams werden gegen alle anderen Teams in derselben Gruppe antreten.
-
 3. Rankings werden für jede Gruppe durch eine Sortierung von den Ergebnissen anhand der Performance jedes Teams an den folgenden Prioritäten ermittelt:
-   1. Am meisten gewonnene Partien.
-   2. Hat ein größeres `{(Anzahl an gewonnenen Maps) - (Anzahl an verlorenen Maps)}`.
-   3. Am meisten gewonnene Maps.
-   4. Hat ein größeres `∑{(gesamtes Ergebnisdifferenz) / (maximales Ergebnis)}`.
-   5. Gewinner von Rückspielen.
-
-4. Die besten 2 Teams in der Gruppe kommen in die Double-Elimination Etappen.
-
-5. Die folgenden Etappen sind Double-Elimination Etappen. Dies bedeutet, dass der Gewinner normal in die nächste Etappe kommt und das andere Team in die Verliereretappe kommt.
-
+   - Am meisten gewonnene Partien.
+   - Hat ein größeres `{(Anzahl an gewonnenen Maps) - (Anzahl an verlorenen Maps)}`.
+   - Am meisten gewonnene Maps.
+   - Hat ein größeres `∑{(gesamtes Ergebnisdifferenz) / (maximales Ergebnis)}`.
+   - Gewinner von Rückspielen.
+4. Die besten 2 Teams in der Gruppe kommen in die Doppeleleminierungsetappen.
+5. Die folgenden Etappen sind Doppeleleminierungsetappen. Dies bedeutet, dass der Gewinner normal in die nächste Etappe kommt und das andere Team in die Verliereretappe kommt.
 6. Basierend auf [dieser Grafik](https://puu.sh/bUq5V/f1066103b0.png) teilen sich die Etappen wie folgt auf:
```

Comment:
```suggestion
6. Basierend auf [dieser Grafik](/wiki/shared/stages-visual.png) teilen sich die Etappen wie folgt auf:
```
Sorry, but I overlooked this the last time. The link is also wrong in the 2014 article (2013 doesn't have it).